packages/relay/src/lib/eth.ts

```diff
@@ -260,6 +260,10 @@ export class EthImpl implements Eth {
     this.debugServiceImpl = new DebugService(mirrorNodeClient, logger, this.common);
   }
 
+  private shouldUseCacheForBalance(tag: string | null): boolean {
+    return !CommonService.blockTagIsLatestOrPendingStrict(tag) && !CommonService.isDevMode;
+  }
```

Comment:
Add a comment as below to improve readability.

```suggestion
  private shouldUseCacheForBalance(tag: string | null): boolean {
    // should only cache balance when is Not latest or pending and is not in dev mode
    return !CommonService.blockTagIsLatestOrPendingStrict(tag) && !CommonService.isDevMode;
  }
```